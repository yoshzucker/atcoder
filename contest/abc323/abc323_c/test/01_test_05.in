99 94
2200 800 1300 2000 2100 700 1300 600 1900 1900 900 2200 2300 1500 1400 1900 1700 1600 800 2100 1200 2200 1600 1700 500 2500 1200 2100 1100 2300 1700 1400 1600 900 2400 800 1800 2300 1900 1400 1500 600 2400 800 800 1500 800 1800 1600 2400 2200 800 1100 2400 1600 700 1600 900 1200 2300 1300 1200 1900 900 900 2500 1000 2000 2400 2100 1500 800 1300 1800 1400 2300 1400 1800 1500 1500 1500 1700 1900 1400 2000 1900 2100 1400 1200 700 600 1400 1500 1600
xxooooxoxxoxoxxxoxxoxoxoxxoxxooooxxooxxooxoxxxoxxoooxoooxoxxoxxxxoxoxxooxxxxoxoooxxoxooxoxxoox
xooooxxoooxooxoxooxxooxxxxoooxxxxoxooxoooooooooxxoooooxxxoooxoxxxxxxoxxoxxooxxxxxoxoooxooxxxox
xoxoxxxoxxxxoxxoxxxoxoooxxxxxxxooxxxxoxooxxoxxxoxooooxooxoxoxxoxooxoxoxxxoxxxooxooxooxxxoxxooo
xooxxooooooxxxoxoxooxooooxxoxoooooooxooxooxxooxoxxxoxxxxxooxxxxooxoxoooxxxooxxxoooooxxxxxxxoox
xoxoxoxoxxxxxxoxooxoxxoxxooxxoooxoxooxooxoxxooxooxoxooooxoooxxoxoxxxxxxooooxoxooooxoxoooxooxxo
xxoooxoxxoxxooxxxxxooooxoxoxxoxoxooooooxxxoxoxxxooxoxxxxooxxxxoxooxxoxoxxxxxxxxxooooxxxooooooo
oxooxxoxooxoooxxoxxxxoxxoxoxxxxxoooxoooooxoxxxooooxoxxxoxxoxoxxoxxxxoxxxooxxooxxoxxooxxoxoxoxo
ooxxoooxoxoxoooxoooxoxoxoxoxxxoooxoxxxxooooxxoxxxxoxoxoooooooxoooxxxoxxoxooxxooooxooxoxoooxxxx
xxxxxxooxoxoxxxxoxxxoxoxxxxxooxxxxxoooxooxoooxxxxooxoooxooxxoxoxxooooooooxxoxoxxxxoxxoxoxooxxo
xooxxooooxxxxoxooxxoxoxxxxxxxoooxooxxoxxxxxxoooooxoxoooxxxoxxooxooxoooxxxoxxoooxooooxoxxoooxoo
xxoxoooooooxoxoxoxxxxooxxooooxooxxxxooooxoxxoxxxxoxooxxxoooxxxooxoooxxxoooxooxxxxxoxxoooxxooxx
oxxooxoxxxoxxoxoxxoxxooxxxxooooxooxoxoxooxooxoxoxoxoxxxoxxooxxxxoxxxoxoxxoxxxxoxxxxooooxoooxoo
xoooxxxoxxxxoooxooxxooooxoxoxxxxooooxooxoooxooooooxxoxoxxxxooxooooxoooxoxooooxoxxxxooxxxoxoooo
xxoxooooooxooxooxxxxooxoxxxooxxoooxxxooxxxoxxooxxoxxooxxooxxxoooxxxooxxxxooxxxxxooxoxxoxxooooo
xoxxoxooxxoooooxoxxoxooxxoxxoxooooooxxxxoxxxxxxxxoooxoooxoooooxxoxoxxooooxoxxxxooxxxxoxoooooxx
oooooooooooxxxoooooxooxoooxooxxooxxxxxooxoxxoxxooxxoxxooxxoxxoooxoxoxoooxooxxoxooxoxooooooxoxx
oxxooooxxoxoxooxoxxxoxooooxxxoooooooooxoxooooooxxxxoxoxooxooxoxoxxxooxxooooooxxxoxxoxooooxxoxx
xoooxoxooxxoxoxoxoxxxoxoooooxxxoooxxoxxoxoxxooooxxxxxoooxooxoooxooooxxxooxooxxxxooxoooooxooxox
xooooxxxxoxooxoooooxoxxxxoxoxoooooooxxooxxxoooxxxooxooooooxoxxxxxoxoooooxxxoxooxoxxoxxxoooxxxo
xooooxoxoxxoxxxxxoxxxxxxxxxxxoxxxxxoxxxoooxxxxooxoxooxxxxxxxxxxxxxoxooxxoooxooxoxooxoooxxoxxxo
xxxoxooxxooxooxoxxxxxxoooxxxxoooxooxxxoxxxoxxoxooxxxxoxxxooxooooxxxxxoxooooxooxxoxoooxxooxxoxx
oooxxoxoooxoxxoxoxoxoxooxooooxooxooxoooxooxxxoxxxxxooxoxxoxxxoxxxxoxoxoooxxoxxooooxxxoooxooxxx
ooooooxoxooxoxxxxxoooxxooxxoxxxxxxxxoooooxoxxoxxoxxxxxooooxxoxxxoooooxoxoxxxoxxoxoxxxoxxooxxox
xoxxxoooxoxxoxxoxxoxxxoxxxoxoxooooxxooooxoxoxoxxxoxoooooooxxooxxxxxoxoxxxxxooxoxoxxxoooxoxxxoo
oooxxxooxooxoxoxxooxoxxoxxoxxooxoxxxxxoxxxoxoooxxxoxoxoxxooooooooxxoooxoxooxxooooxoxoxoxoxoooo
oxoxxoxooxxooxoxoxoooxxxxoxxooxxooxxxooxooxooxoxxoooooxoooxooxxxoxxxxoxoxoxxoxxxoooxooxxxoooox
xoxoooooxoxoxxoxxoxxxooooxoxooxxooxxxxxxooxoooooxooxooxxoxoxoxoxoooxxoxoxoxoooxooxoxoxxxxooxxx
xxoxxooxxoxoxxooxoooooooxoxxoxxoooxxxxxxoxoxxooxoooxooxooxxooxoxoxxxoxoxxxoxxxoxxxoxoxoxoxooox
xoxooxxoxxoooxxoooxooxxoxxxooxoxxxoxxxxoxxxxxooooxooxxxoooxooxoxoooxoxooxoxxxooooxxooxxxoxxoox
xooxxxxxoxxxxooxooxooxxxooxxooooxoooxxoxxxxoxxxxxoxxxxoxoxoooxxoooxoxxxoooxxxoxxxxxooxoxxooxoo
oxxoxxooooooxxoxoooxxxxooxooxxxoooxooooxoxxoxxxxoxxxxoooooooxoxoxoooxoxoxooxoxoxooooxxoxoxxxxo
oxxxooxxoxooxooxoooxxoxoxxooxoxxoooxoooxxxoxoxxoxooxoxxoxxoxxoxooxxxxxoxoxxoxooxxxooxooxxooxoo
xoxoxoxxooxxxoooooxxoxoooxoxxxooxxxooooxxooxoxxxxxoxoxoxoxooxxxxxoxoxoxxxxxxxxxooxooxoxxoooxox
oxooooooxxooxxxoooxxxooooxoooxxoooxxooxooxxxxxxxxxoxooxxooooooxxxxoxxxoxxxxoxxxxoxoxooooxoooxx
ooxxxooxxooxoxxoxoxxooxoooxxoooooxxoxxxxoxxooooxoxoxoxxxoxxoxxoxoxoooxooooxooxooooxxooxoooxoxo
xoxoooxoxoxoxxxxoooxoxoxooooxxxxxxxxxxxxoxxoxoxxxxoooxoxoxxxxxxooooxoxoxxoxxxooxxxoxxxoxxxxxxx
xxxoxooxooxoooxxxxoxooxoxxooxxxoxoxxxoxxoooxooxoxoxoooxoxxoxoxooxxooooxxoxxxooxoooxooxooooxxox
oxxxoooxxooooxoooxoooxxxoxxxxoxxoooooxoxxxooxxoxoxoxooxxoxxoxxoxxoooxxxooxoxoooxxoxxooxoxooxoo
oxxxooxoxxxoooxxoooxxooxooxxoxoxxooxoxxxxxooxooxoxoxoxoxoooooxxxxoxxoxxxxoxxxxoxxxxoxooxxxooxx
oxxooooxxoxoooxxxoxoxxxoooxoooxxoooxoooxxxooxooooxooxxooooxoxxxoxooxoxoxxxxxoxoxooxooooxxxooxo
xxxxxoxoxoxooxxxxxxxxxooooooxoxoxoxxoxooxxooxoxxoxxxoxoxxoxooxxoxxoxxooxxxooxxxooxoxxoooxooxxx
xoxxooxxxxxoxoxoxoxxxxxxoooooxxxxoxxxoxooxxoooxoxoxoxxoxxxxoxxoxxxoxoooxoooxooxoooxoooooooxxoo
xxxxoxooxoxoxoxooxxoxoxoxxxxoxoooooxxoxoxoxxxoooxooxxoxooooooxxxoxoooxxxxxxooxxxxoxoxooxoxoxoo
xooxoooxxoxooooxoxxoxxxooxxxooxoxoxooxxxooxooxxxxxooxoxoxooxxxxxoxooooxooxoxxxoooxxoxxoooxxxox
oooooxoxxxoxxxxxooxxoxxooxoxoxooxoxxxooooooooxxoxoxxoooxoxxooxoxxxoooooxoxoxxxoooooxxxxxxxooox
oooxxooxxooxxxxxxxoooxxoooxxooxxxxoxooxoxxoxoooxooxoooxoxooxxoxxxooxxoxxoxoxxxxxxxxoxoooxoxxxx
oooxxxooxoxoxxooxxxxooxooooxxxxooxxoxxxxxxxxoooxoooxxxxxxoxxoxxoooxooxxoxooxoxoxooxxxxxxoxxooo
xoxooxxxxxxxxoxooxooxxooxooxoxxoxxxxooxoxoxxxoxxooxxooxxooxxoxooxoxxoxooxoxooooxxoxxoxxxxoxooo
ooxxxxxooooooxooxxoxooxoooxxxoxxooooxoxxoxooooxoxxxxxooxoooooxxoooooxoxoxxoooxoooxoxxoooxooxoo
oxooooxxxxxoxoxxoooooooxoxoxxoxoxxxoooxoxxoxxoxxxooxoooooooxoooxxxxxooooooxxoooooxxooooxoxxxox
oxoxxxoxoxxoxoxoooxxxxooooxxoxxoxxoxoxoxxxooooxxoxoxxxxoxoxxoxxxoooxoxoxoxxxooooxxxoxxoxoxxxxx
xoxoxxxxxooooxoooxxxooxxxoxooxxxoxoxxxoxxoxoxxxooxxoxxoooooooooxooooooxxxxooooxoooooxoxooxxxxx
xooxooxoxxxoxoooooooooooxxoxoxxoxooxoxxxxoxoooxxxoooxoxxxoooxoxxoxoooxooooxxxxxxoxoxoooxoooxox
oxxooxxxooxoxxooxxoooxxoxoxxxxoxxxxooxoxoxxoxxxxoooxxoooxxoxoooooxoxooxoxxxooxxooooooxxoxxoxox
oxxxooxooxxxxooooxxoooxxoxoxxooooxxoxxxxooooxooxoooxxxooxxoxxoxxxooxoooooxxoooxxoxoxooooxxxxxx
oxxoxoooxoxooooxxxxxxoxoxooxxoooxxxoxxxoxxoxoooxooxoxooxoxxxxooooooooxoxooooooxxoxxoxooxoxxooo
oxoooooxooxooxoxxxoooooxoxxooxoxoxoxxoxxxooxoxoooxxxoxoxoxxxoxoxoxoxoooxxoxxxxxxxxxxxxoxooxxxo
xoxoxxoooooooxxxoooxxxxoooxxooooxxooxoxxoxxxxxoxoxxxoxxxoooxoooxooxxooxxooxxoxxooooooxoxxxxxxo
oooxooxoxxoxoooxxxxxxoxooooxoooxooooxoxoooxxxxxxoooxooxoooxxoxxxoxxxoxooooxxxxxxoxxooxxooooooo
ooxoxoxxoxxxxxooxooxoooxoxoxxooxxxxxxoxxoooooxoxooxxooooxxxxoooxoxxoxxooxooxooxoooxxoxoooooxxo
xoxoxxxoooxxxoxxoxoooxoxxoxxoxxooxxxooxooxoxxxxooxooxoooxxxxxxooxxxoxxxxxoxoxoxxxxooxoxxxxooxo
ooooxxxxxxxoxxxxoxooxooooxoxxooxooooxxxooxxxoxxxxooooooxxoooooxoxxoooxooooxxooxooxxxoxoxoxxoxo
xxxxoxoxooooxxoxxxxxxxxxxooooxoxoxoxoxoxoxoxooxoxxxooxxxxxoxxxxoxxxxxoxoxxoxxoxxooooxoooxxoxox
oooxxxoxxxxxxxoxoxxoxoxoxoxooxxoooxxoxxoooooooxoxxooxxoxxxxoxoooxxoxxxoxxxxooxooxoxooxxooxooox
xoooxxoxooxxoxxoxxooxoooxoxxxxxxxxxxoxxooxoxxxxxoxoxxooxoxoooxoxoxooooxooooxoxoxoxoxxooxooxoxo
oxxoxoxoxooooxxoxoxoxxxoooxxxoxxxoxoooxxxoxxooooxxxoxooxooooxxxxooooxoxoxoooxxoooxxxoooxoooxox
xooxoxxooooxxooxxoxoooxooxoxxoxoxoxooxoxoxxooxoxooxoooxoxoxxoxxxooxoxxxxoxxoxxooooooxxooxoxoox
xoxoxxxoxxoxxxoxxxxooooxoxoxxxxoxxxxoxoooooooxxoooxoxooxooxxoxxoooxooxxxooooxoooxooxooxoxoxxoo
oooxooxooxxxoxoxoxoxxxooooxxooxoxxxoxoooxxoooooxoooxxoxoxxoxoxooxxoooxooooooxoxooxxxoxoxooxooo
xxoxoxooxoxoxxxxxxooooxxxxxoxxxxoxxxxoxxxooooxooxxxxooxooxoxoxxxxoxxooxoxxoxoxxxooooxxoxxoxxoo
oooxooxxxoooxxxxxoooxxxooxooxxoxxxxxoxoxooooooxoooxxoxxoooxxooxooxxoxxxoooxooooxxooooxoxooxooo
xxoooooxoxxooxxoxxxooooxoxoxoxoxxxooxoooxxoxooxxxxxoxxoxxxooxxxoxxxxxxxxxoooxxxxoooxooooooooox
xoxooooxoooxxxxxoxxxxoxoxoxooxxooxooooooxooxxoxoooxooxoxooxxxoxxxxxxoooxoxooxoooxooxoooxxxxoxx
xxxxoxoooxxxxooxooxxoxxxoooxooxooxooxxoooooxxxooxxxxoxoooxooxxoxoxxoxoxxooxoxoxooxoxoxxxoxoxxx
ooxxoxxooxooxoxooxxoxooxoxxoooxxxxxoxoxoxxoxooxooxxxoooooooxooxoxxxoxoxooxxxxoooxxxxxoooooooxo
xoxoooxxooooxoooxoooxoxxooxooxoxoxoooxoxxxxxxxxxxxxoxxoxxxxxxoxoooxooxoxoooxoxoooxoxxoxoxxoxox
oooooxxoxoxooxooxooxooxxxxxoooxoooxxxxooxoxxxoxooxxxooxxooxxoxxoooxoxxooxxxxxooooxoxoxxxoxxxxx
oxoooxoooxooxoxoxoxooxoxooooxoxooooxxoxoooooxxoxoxoxxxxxxoxxxoxooxxxxooxxxxxoxoxxoxxoxoooxxoxo
xoxooxooxxxooxxoxooxooxoxoxxooooxxxxxooooxxooxoxoxxoxooooxxxoxoxxxooooooxoxxxoxoooxoooooxoxooo
ooxxxxoxoxooxoooxxxxooxxoxoxxoxxooxxxxooxoooxoxxxxxoxoxxxxxxxxooxooxoooooxxoxxooxxxxxooxoooxoo
xxoxooxxoxooooxxooxoxooxxxoooxxoooooxxooxxoooooxoxxooooooxoooxoxoxoxooxxoxxoxooxxxxxxxooxxoxox
xoxxxooxxoxxooxoxxoxxxxooooxxxxxxxooxoxxxxoxxoxxoxooxxxooxxxxxxxxooxxooxxooxoxxxoxoooooxoooooo
oxxoxoooxxxooxooxoxxoxxoxxoxooxoxoxoxxoooxxxoxxoxxooxooooxooxxxxxxooxoxxxxxxooooxxooxoxoxxxooo
ooooooxxxxoxxxxooxxoxooxxoxxxoxoxxoxooooxxxxooooxooxxxxxxxxxxooxoooxxoxoooxoooxxxxoxoxxxxoxxoo
xoxxxxxoxxooooxooxxooxxoooooxxoxoxooxxoxxooxoxoxooooxoxxxooxooxoxoxxoxxxoxxooooxoxooxxoxoxxxoo
oxxooxxoxoxoxoxoxooxxoooxoxoooooxoooooxooxoxoxxxxooooxooooxooxxxooxoxxoooxxooxxxxoxoxxoxxxxxxx
xxxoooxxoxooxxoxoxoxooxoxoooxoxoooooxoxooxxooxxxxooxxoxxxxxoxxxxxxxoooooxoxoxooxxoxxoooxxoooxo
ooxoxxxooxxxoxxxxxxxooooxooxooxooxoxoooxooxxxxoooxxoooooxoooxoxoxxooxxoxooxoxxoxoxooxxooxoxxox
oooxoxxoxooxxooxooxooxoxoxxxoxxxooooxooooxxooooxoxoxooxxxoxxxooxxxxoxoooxxooxxooooxooxoooxoooo
xoooxxoxoooooxoxxxoooxooooxooxxooooxxxxoooxxxxxoxooxxxxooooooxoxxxxooxxoxxooxoxxoxooooooxooxxo
xxxooooooxooxxoxxxxoxxxxooooooxoxxoxxoxooxoxxxxxxoxxooxxxooooooooxoooooooooooxxxxoooxxooxxooox
xxxxoxoooooxxoxooxxoooooxoooooxxxxxxxooxxooxxooxxxxxoxoxooxxxxoxoooxxxooxxoxxoxxooxxoxxxxxxoxx
xxoxoxxxoxxooxoooxxoxoxxoooxoxoxoooxxooooooooxoxxxxooxoooxxoxoxxoxoooooxoooxxoxxooxoxoxoxxoxox
xxoxxxooxxooxxoooxoooooooxxooxxxoxoxooxoxoxoooxoooxxxoooooxoxooooxoooxooxoxooxoxxoooxoxxooxxoo
xooxxxoxoxoooxxxoxxxxoxoxoxxxoooooxoooxxxoooxoxooooooxxxxoooxoxooxxoxxoxxooxxxxxooxoxooxoooxxx
xoxxxooxxxxxooooooooxxoxoxooooxoooxoxoooooxxoooooxxxxxxoxoxoooooxoxxxooxoxxoooooxxxxxxxooooxoo
oxxooxoooxxoxxoxxooxxxxooooxooxooxoxooxooooxxxooxoooooooxxxxxxooxoxoooooxooxxoxxxxxooooxoxoxxo
xoxxxxooxxooxxoxxoxooxxxooxoxoxooooxoooxxxxxxxooxoxxxxooxoooooooooxooooxxooxxxoxxxxooxoxxxxoxo
oooxxxxxoxoxooooxoxoxoooxooxoxoxoooxoxoooxxxoxoxxxxoxoooxxoxoxxxoxooooooxxxoooxoooxxooooxoooxo
