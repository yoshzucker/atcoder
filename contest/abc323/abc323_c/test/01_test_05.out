8
5
10
7
7
8
8
5
9
5
7
9
1
7
7
2
2
5
2
15
10
7
10
10
5
7
7
7
7
11
4
6
10
7
4
16
5
5
10
1
12
7
6
9
5
12
11
8
0
1
11
4
5
6
5
2
8
6
6
6
11
4
12
8
5
5
6
6
2
12
4
7
3
8
5
7
8
6
2
10
4
11
8
7
6
3
4
5
3
3
2
10
4
1
7
5
2
6
2
